### ISSUE TITLE
Add the ability to utilise all fields's value of the node in callbacks

### ISSUE THREAD

**Issue by AnsonHwang86** [neo4j/graphql#1497]
**Is your feature request related to a problem? Please describe.**
Could you also add the ability to return the value of all field along with the added/updated value? such as
```
"products": [
				{
					"listings": [
						{
							"id": "8264b791-4sd8-411d-ba32-39f6277a8605",
							"sku": "ssd",
							"sellingPlatform": {
								"sellingPlatformCode": "XX"
							}
                                                 ... all other fields's values in the node
						}
					]
				}
			]
```
My use case is to concat value between different fields. 
Currently, we can get only the new insert value or updated value.
https://github.com/neo4j/graphql/pull/1361

**Comment by AnsonHwang86** [comment]
yes, I want to get all fields of updated/created node. Why the callback run before database execution. I'm so confused about that.

**Comment by Tenrys** [comment]
This ties in with https://github.com/neo4j/graphql/issues/652 I would say, but I am of the same opinion. It would be nice to be able to run a function after database execution, and precise in the directive which fields we would like to use from the concerned node.

**Comment by AnsonHwang86** [comment]
Yes, I once used Dgraph for a little time. Yes, they offer two kind of callback, one before execution and one after execution. Both two would help to keep the data consistency.
Hope neo4j/graphql can implement this too.

**Comment by darrellwarde** [comment]
Closed as duplicate of #2265.
